app/models/external_service/url_service.rb

```diff
@@ -23,6 +27,10 @@ def edit_url(receiver, request = nil)
 
   private
 
+  def formio_form?
+    URI(location).host.try(:ends_with?, "form.io")
```

Comment:
Is there a reason you use `include?` above and `ends_with?` here?

Reply:
Not a good one! I think I like the tighter `ends_with?` here, so I’ll change the above invocation. The URLs here are things like `https://nucore-development-12ea0e74.form.io/labitsupport`, so the host should always end in `form.io`.